README.md

```diff
@@ -161,7 +161,7 @@ yarn build:docker
 
 ```bash
 export VERSION=`cat package.json | jq -r .version`
-docker run --rm -it -p 8080:8080 substrate-api-sidecar:$VERSION
+docker run --rm -it -p 127.0.0.1:8080:80 substrate-api-sidecar:$VERSION
```

Comment:
- `:80` is usually used for UIs btw. You may pick 8000, 8080, or 3000 by default.
- While `127.0.0.1:8080:80` is safer, I would recommend simply `8080:80` which will work 'easier' for most people. You may point to an article about making docker infrastructure safer so we don't need to do it here.

I suggest to add a docker version for the non dev that shows up first in the doc:
```
docker run --rm -it -p 8080:80 w3f/substrate-api-sidecar
```

Reply:
Where is the name `w3f/substrate-api-sidecar` specified?

Reply:
For the point about not setting it `127.0.0.1`, what is the downside? I was thinking it would make other, non-docker, documentation easier to reason through since the it would always be the same/

For your last point about the docker version for non dev, could we just default to having the build command name it `substrate-api-sidecar`? I think it might be confusing to have two different naming schemes since we would then need to have different commands just to build images with different names.

Reply:
I guess maybe a better question to ask is what is the utility of having the image named with the version? Is this a common idiom?

Reply:
> Where is the name w3f/substrate-api-sidecar specified?

I found it here: https://hub.docker.com/r/web3f/substrate-api-sidecar

> For the point about not setting it 127.0.0.1, what is the downside? 

You should not assume that people use Docker on their local machine. In production, this docker container will run on a server. This 127.0.0.1 means that only services running on the server will be able to access sidecar.

> For your last point about the docker version for non dev, could we just default to having the build command name it substrate-api-sidecar?

Yes, you can! This is my second suggestion. The naming scheme is not up to us, this is docker standard: `<user>/<image>:<tag>`.

Reply:
Great - removed the localhost ip and and updated the build and run command to not use version for simplicity.